workers/unity/Assets/Gdk/Core/Error/ForwardingDispatcher.cs

```diff
@@ -0,0 +1,85 @@
+using System.Collections.Generic;
+using Improbable.Worker;
+using UnityEngine;
+
+namespace Improbable.Gdk.Core
+{
+    public class ForwardingDispatcher : ILogDispatcher
+    {
+        private Connection connection;
+
+        private readonly LoggingDispatcher loggingDispatcher = new LoggingDispatcher();
+
+        private readonly LogLevel minimumLogLevel;
+
+        private static readonly Dictionary<LogType, LogLevel> LogTypeMapping = new Dictionary<LogType, LogLevel>
+        {
+            { LogType.Exception, LogLevel.Fatal },
+            { LogType.Error, LogLevel.Error },
+            { LogType.Assert, LogLevel.Error },
+            { LogType.Warning, LogLevel.Warn },
+            { LogType.Log, LogLevel.Info }
+        };
+
+        public ForwardingDispatcher(LogLevel minimumLogLevel = LogLevel.Warn)
+        {
+            this.minimumLogLevel = minimumLogLevel;
+
+            Application.logMessageReceived += LogCallback;
+        }
+
+        private void LogCallback(string message, string stackTrace, LogType type)
+        {
+            if (type == LogType.Exception)
```

Comment:
Does this mean that warning and errors are silent? Wouldn't it be better to expose the min-loglevel for forwarding arbitrary unity logs as a config parameter?